bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -63,10 +63,24 @@ impl LockTime {
     }
 
     /// Returns true if this [`relative::LockTime`] is satisfied by `other` lock.
+    #[deprecated(since = "0.30.0", note = "Use is_implied_by() instead")]
+    pub fn is_satisfied_by_lock(&self, other: LockTime) -> bool {
+        self.is_implied_by(other)
+    }
```

Comment:
The function was not available in a published release, so no point of deprecating - just remove it.

Reply:
Yes, you are right - thanks.